Allows configuring TLS backend (#386)
tealdeer-rs/tealdeer: docs/src/config_updates.md

    [updates]
    archive_source = https://my-company.example.com/tldr/

### `tls_backend`

> An advance option. Specifies which TLS backend to use. Only modify this if you encounter certificate errors.

Reviewer: What do you think about the following?
```suggestion
Specifies which TLS backend to use. Try changing this setting if you encounter certificate errors.
```

Available options:

> - `rusttls-with-native-roots` - [Rustls][rustls] (a TLS library in Rust) with native roots
> - `rusttls-with-webpki-roots` - Rustls with [WebPKI][rustls-webpki] roots

Reviewer: Typos:
```suggestion
- `rustls-with-native-roots` - [Rustls][rustls] (a TLS library in Rust) with native roots
- `rustls-with-webpki-roots` - Rustls with [WebPKI][rustls-webpki] roots
```
